### ISSUE TITLE
False positive for hints matched in method calls

### ISSUE THREAD

**Issue by duckmatt** [fsprojects/FSharpLint#36]
```
type Bar() =
    static member SomeMethod(foo: bool) = ()

Bar.SomeMethod(foo = true)
```

This currently warns with `x=true can be refactored into x` because it's interpreting the named parameter as checking for equivalence rather than being assigned a value
